roxFile/createroxfile-service.ts

```diff
@@ -16,17 +23,17 @@ function createFileIDField(fileIDFieldName: string, response: any, instance: PH.
   }
 }
 
-function generateTitleWithDataType(title: string, dataPath: string): string {
+function generateTitleWithDataType(title: string, fileName: string): string {
   if (!title.trim()) {
-    const dataPathSegments = dataPath.split("/");
-    return dataPathSegments[dataPathSegments.length - 1];
+    const dataPathSegments = fileName.split("/");
+    return dataPathSegments.last();
   }
 
   const splittedTitle = title.trim().split(".");
 
   for (const titlePart of splittedTitle) {
     if (titlePart) {
-      return titlePart.trim() + dataPath.substring(dataPath.lastIndexOf("."), dataPath.length);
+      return titlePart.trim() + fileName.substring(fileName.lastIndexOf("."), fileName.length);
```

Comment:
This breaks with files like `test.test.ts` which is bad I think. It's maybe not that common but will then suddenly modify the file name magically.
Also think about files like `.htaccess`

Reply:
Now just the first segment is replaced by the title. So if the title is "hello" and the filename would be test.test.ts, the resulting roxfile name would be hello.test.ts.
Analogical for .htaccess, the roxfile Name would be hello.htaccess.